ddtrace/_trace/trace_handlers.py

```diff
@@ -956,10 +1133,10 @@ def listen():
         "flask.jsonify",
         "flask.render_template",
         "asgi.__call__",
-        "asgi.websocket.close_message",
-        "asgi.websocket.disconnect_message",
-        "asgi.websocket.receive_message",
-        "asgi.websocket.send_message",
+        "asgi.websocket.close.message",
```

Comment:
if you add these events to the list for `context.ended.{name}` below which calls `_finish_span`, then you can remove the `, ctx.span` from `with ... as ctx, ctx.span:` calls.

We want to try and deprecate `ctx.span` entirely, and make sure the ExecutionContext does not have any relationship or knowledge about spans.